<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>716</width>
    <height>569</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>540</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Deformation model</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="view" name="graphicsView">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>1</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="sizeConstraint">
         <enum>QLayout::SetMinimumSize</enum>
        </property>
        <item>
         <widget class="QLabel" name="concreteClassLabel">
          <property name="text">
           <string>Concrete class</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="concreteClassComboBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="drawSectionBox">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
          <property name="title">
           <string>draw section</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>5</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QPushButton" name="newButton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="toolTip">
              <string>New section</string>
             </property>
             <property name="text">
              <string>New</string>
             </property>
             <property name="checkable">
              <bool>false</bool>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="drawLineButton">
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="toolTip">
              <string>Draw line</string>
             </property>
             <property name="text">
              <string>Line</string>
             </property>
             <property name="checkable">
              <bool>false</bool>
             </property>
             <property name="default">
              <bool>false</bool>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="drawRectButton">
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="toolTip">
              <string>Draw rectangle</string>
             </property>
             <property name="text">
              <string>Rect</string>
             </property>
             <property name="checkable">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="divideButton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="maximumSize">
              <size>
               <width>40</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="toolTip">
              <string>Divide the section</string>
             </property>
             <property name="text">
              <string>Divide</string>
             </property>
             <property name="checkable">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="reinforcementClassLabel">
          <property name="text">
           <string>Reinforcement class</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="reinforcementClassComboBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="drawReinforcementBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>50</height>
           </size>
          </property>
          <property name="title">
           <string>draw reinforcement</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>5</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>5</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_10">
             <item>
              <widget class="QPushButton" name="newRButton">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximumSize">
                <size>
                 <width>60</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="toolTip">
                <string>New reinforcement</string>
               </property>
               <property name="text">
                <string>New</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="drawPointButton">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximumSize">
                <size>
                 <width>60</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="toolTip">
                <string>Draw reinforcement</string>
               </property>
               <property name="text">
                <string>Point</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_9">
             <property name="spacing">
              <number>5</number>
             </property>
             <property name="leftMargin">
              <number>5</number>
             </property>
             <property name="topMargin">
              <number>5</number>
             </property>
             <property name="bottomMargin">
              <number>5</number>
             </property>
             <item>
              <widget class="QSpinBox" name="diameterSpinBox">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximumSize">
                <size>
                 <width>45</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>32</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="diameterLabel">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Diameter</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="forceBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>110</height>
           </size>
          </property>
          <property name="title">
           <string>Force</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="NLabel">
               <property name="minimumSize">
                <size>
                 <width>30</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>30</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>N</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="NLineEdit">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximumSize">
                <size>
                 <width>60</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="baseSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="NUnitLabel">
               <property name="minimumSize">
                <size>
                 <width>20</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>kN</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="MyLabel">
               <property name="minimumSize">
                <size>
                 <width>30</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>30</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>My</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="MyLineEdit">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximumSize">
                <size>
                 <width>60</width>
                 <height>16777215</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="MyUnitLabel">
               <property name="minimumSize">
                <size>
                 <width>20</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>kNm</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="MxLabel">
               <property name="minimumSize">
                <size>
                 <width>30</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>30</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Mx</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="MxLineEdit">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximumSize">
                <size>
                 <width>60</width>
                 <height>16777215</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="MxUnitLabel">
               <property name="minimumSize">
                <size>
                 <width>20</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>kNm</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="calculateButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Calculate</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="commandWidget" name="commandTextEdit">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>716</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuDraw">
    <property name="title">
     <string>Draw</string>
    </property>
    <addaction name="actionNewSection"/>
    <addaction name="actionLine"/>
    <addaction name="actionRectangle"/>
    <addaction name="separator"/>
    <addaction name="actionNewReinf"/>
    <addaction name="actionPoint"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuCalculate">
    <property name="title">
     <string>Calculate</string>
    </property>
    <addaction name="actiondivide"/>
    <addaction name="actionCalculate"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionFit_to_section"/>
    <addaction name="actionZoom_in"/>
    <addaction name="actionZoom_out"/>
    <addaction name="actionPan"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuDraw"/>
   <addaction name="menuCalculate"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionFit_to_section"/>
   <addaction name="actionZoom_in"/>
   <addaction name="actionZoom_out"/>
   <addaction name="actionPan"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="resourse.qrc">
     <normaloff>:/Icon/fromExcel4.ico</normaloff>:/Icon/fromExcel4.ico</iconset>
   </property>
   <property name="text">
    <string>Import points</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="resourse.qrc">
     <normaloff>:/Icon/toExcel4.ico</normaloff>:/Icon/toExcel4.ico</iconset>
   </property>
   <property name="text">
    <string>Export points</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionRectangle">
   <property name="text">
    <string>Rectangle</string>
   </property>
  </action>
  <action name="actionNewReinf">
   <property name="text">
    <string>New reinforcement</string>
   </property>
  </action>
  <action name="actionLine">
   <property name="text">
    <string>Line</string>
   </property>
  </action>
  <action name="actionPoint">
   <property name="text">
    <string>Point</string>
   </property>
  </action>
  <action name="actionSet_concrete_section">
   <property name="text">
    <string>Set concrete section</string>
   </property>
  </action>
  <action name="actionSet_reinforsement_section">
   <property name="text">
    <string>Set reinforsement section</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="text">
    <string>Options</string>
   </property>
  </action>
  <action name="actiondivide">
   <property name="text">
    <string>Divide</string>
   </property>
  </action>
  <action name="actionCalculate">
   <property name="text">
    <string>Calculate</string>
   </property>
  </action>
  <action name="actionNewSection">
   <property name="text">
    <string>New section</string>
   </property>
  </action>
  <action name="actionFit_to_section">
   <property name="icon">
    <iconset resource="resourse.qrc">
     <normaloff>:/Icon/fit1.ico</normaloff>:/Icon/fit1.ico</iconset>
   </property>
   <property name="text">
    <string>Fit to section</string>
   </property>
  </action>
  <action name="actionZoom_in">
   <property name="icon">
    <iconset resource="resourse.qrc">
     <normaloff>:/Icon/zoom_in1.ico</normaloff>:/Icon/zoom_in1.ico</iconset>
   </property>
   <property name="text">
    <string>Zoom in</string>
   </property>
  </action>
  <action name="actionZoom_out">
   <property name="icon">
    <iconset resource="resourse.qrc">
     <normaloff>:/Icon/zoom_out1.ico</normaloff>:/Icon/zoom_out1.ico</iconset>
   </property>
   <property name="text">
    <string>Zoom out</string>
   </property>
  </action>
  <action name="actionPan">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resourse.qrc">
     <normaloff>:/Icon/hand.ico</normaloff>:/Icon/hand.ico</iconset>
   </property>
   <property name="text">
    <string>Pan</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>commandWidget</class>
   <extends>QTextEdit</extends>
   <header>commandwidget.h</header>
  </customwidget>
  <customwidget>
   <class>view</class>
   <extends>QGraphicsView</extends>
   <header>view.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resourse.qrc"/>
 </resources>
 <connections/>
</ui>
